apps/bare-expo/android/build.gradle

```diff
@@ -61,7 +62,13 @@ allprojects {
             url "$rootDir/../node_modules/expo-camera/android/maven"
         }
 
+       maven {
+           // Detox is installed from workspace root npm
+           url "$rootDir/../../../node_modules/detox/Detox-android"
```

Comment:
Maybe add detox to the list of symlinks in the app's package.json. This will help prevent a possible future error where detox could have a version mismatch or somn.

Reply:
https://github.com/expo/expo/pull/12893/commits/cc11a20bb22cc962cc9a61645b49c5ec20de26a6